ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -160,11 +174,76 @@ export default class ConfirmPageContainerContent extends Component {
         {this.renderContent()}
         {!supportsEIP1559V2 &&
           !hasSimulationError &&
-          (errorKey || errorMessage) && (
+          (errorKey || errorMessage) &&
+          currentTransaction.type !== TRANSACTION_TYPES.SIMPLE_SEND && (
             <div className="confirm-page-container-content__error-container">
               <ErrorMessage errorMessage={errorMessage} errorKey={errorKey} />
             </div>
           )}
+        {supportsEIP1559V2 &&
+          !hasSimulationError &&
+          (errorKey || errorMessage) &&
+          errorKey === INSUFFICIENT_FUNDS_ERROR_KEY &&
+          currentTransaction.type === TRANSACTION_TYPES.SIMPLE_SEND && (
+            <div className="confirm-page-container-content__error-container">
+              {currentTransaction.chainId === '0x1' ? (
+                <ActionableMessage
+                  className="actionable-message--warning"
+                  message={
+                    <Typography
+                      variant={TYPOGRAPHY.H7}
+                      align="left"
+                      margin={[0, 0]}
+                    >
+                      {this.context.t('insufficientCurrency', [
+                        nativeCurrency,
+                        networkName,
+                      ])}
+                      <Button
+                        key="link"
+                        type="secondary"
+                        className="confirm-approve-content__warning__link"
+                        onClick={() => showBuyModal()}
+                        style={{
+                          color: '#037dd6',
+                          padding: 0,
+                          fontSize: '12px',
+                        }}
+                      >
+                        {this.context.t('buyEth')}
```

Comment:
[nit] might be worth extracting `t` from `context` at the top of the render method:
`const { t } = this.context`

Reply:
done.